.github/workflows/run-integration-tests.yml

```diff
@@ -11,18 +11,12 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  aurora-postgres-integration-tests:
-    name: 'Run Aurora Postgres container integration tests'
+  all-integration-tests:
+    name: 'Run integration tests on all environments'
     runs-on: ubuntu-latest
     strategy:
```

Comment:
You can remove the entire `strategy` section, they are only used if we have the matrix.